build/lib/release.sh

```diff
@@ -419,11 +396,15 @@ function kube::release::package_kube_manifests_tarball() {
   cp "${KUBE_ROOT}/cluster/gce/gci/health-monitor.sh" "${dst_dir}/health-monitor.sh"
   local objects
   objects=$(cd "${KUBE_ROOT}/cluster/addons" && find . \( -name \*.yaml -or -name \*.yaml.in -or -name \*.json \) | grep -v demo)
+  # word splitting by space is intentional and required so disable checkcheck for this line
+  # shellcheck disable=SC2086
   tar c -C "${KUBE_ROOT}/cluster/addons" ${objects} | tar x -C "${dst_dir}"
```

Comment:
how much effort is involved in turning these into arrays? (didn't trace the source)

Reply:
Oh. Thanks for your suggestion. There are ways to change it into array and it works.